src/iguana/algorithms/clas12/ZVertexFilter/Algorithm.h

```diff
@@ -46,6 +47,9 @@ namespace iguana::clas12 {
 
       /// Z-vertex cut
       std::vector<double> o_zcuts;
+
+      /// pids to apply ZVertexFilter to
+      std::vector<int> o_pids;
```

Comment:
Use `std::set` for faster performance, since we expect each element to be unique.
```suggestion
      std::set<int> o_pids;
```
_cf_. `EventBuilderFilter`